### PR TITLE
DEV-7689 Added a use case for audit trail

### PR DESCRIPTION
**Pull request by MikeMcGarry**

### CODE REVIEW

File: examples/use-cases/audit-trail/Change Notification Example.ipynb [hunk 1]
```diff
@@ -0,0 +1,775 @@
+{
```
Comment (rizwansaeed) [inline examples/use-cases/audit-trail/Change Notification Example.ipynb:1]: <p>add story outlining process</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/161/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (rizwansaeed) [inline examples/use-cases/audit-trail/Change Notification Example.ipynb:1]: <p>use format_response_from_dataframe</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/161/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (rizwansaeed) [inline examples/use-cases/audit-trail/Change Notification Example.ipynb:1]: <p>make dataframe</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/161/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (rizwansaeed) [inline examples/use-cases/audit-trail/Change Notification Example.ipynb:1]: <p>make data frame</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/161/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (MikeMcGarry) [reply examples/use-cases/audit-trail/Change Notification Example.ipynb]: <p>Updated.</p>

Comment (MikeMcGarry) [reply examples/use-cases/audit-trail/Change Notification Example.ipynb]: <p>Updated.</p>

Comment (MikeMcGarry) [reply examples/use-cases/audit-trail/Change Notification Example.ipynb]: <p>Updated.</p>

Comment (MikeMcGarry) [reply examples/use-cases/audit-trail/Change Notification Example.ipynb]: <p>Updated.</p>

**Pull request merged**